lib/phlex/component.rb

```diff
@@ -121,34 +71,6 @@ def render_block(new_target, ...)
       @_target = old_target
     end
 
-    def cache_key
-      @cache_key ||= CacheKey.new(
-        self.class,
-        cacheable_content,
-        cacheable_resources
-      )
-    end
-
-    def cache_version
-      @cache_version ||= CacheVersion.new(
-        RUBY_VERSION,
-        Phlex::VERSION,
-        self.class,
-        cacheable_content,
-        cacheable_resources
-      )
-    end
-
-    private
-
-    def cacheable_resources
-      (@_cache == true) ? assigns : @_cache
-    end
-
-    def cacheable_content
-      CacheableObject.new(@_content)
-    end
-
     def assigns
```

Comment:
I think you can remove this method as well :)

Reply:
I left this in because it's quite nice for meta programming components. That said, I'd have a hard time coming up with a real use-case so maybe it's best to remove it for now. It's easy enough to put back in.